src/Sarif.UnitTests/Baseline/ResultMatching/SarifLogResultMatcherTests.cs

```diff
@@ -74,6 +82,44 @@ public void SarifLogResultMatcher_BaselinesTwoSimpleSarifLogs()
             }
         }
 
+        [Fact]
+        public void SarifLogResultMatcher_BaselinesSarifLogsWithProperties()
+        {
+            Random random = RandomSarifLogGenerator.GenerateRandomAndLog(this.output);
+
+            SarifLog baselineLog = RandomSarifLogGenerator.GenerateSarifLogWithRuns(random, 1);
+            SarifLog currentLog = baselineLog.DeepClone();
+
+            string sharedPropertyName = nameof(sharedPropertyName);
+            string currentSharedPropertyValue = Guid.NewGuid().ToString();
+            string baselineSharedPropertyValue = Guid.NewGuid().ToString();
+
+            string uniqueToBaselinePropertyName = nameof(uniqueToBaselinePropertyName);
+            string uniqueToBaselinePropertyValue = Guid.NewGuid().ToString();
+
+            string uniqueToCurrentPropertyName = nameof(uniqueToCurrentPropertyName);
+            string uniqueToCurrentPropertyValue = Guid.NewGuid().ToString();
+
+            baselineLog.Runs[0].SetProperty(sharedPropertyName, currentSharedPropertyValue);
```

Comment:
>currentSharedPropertyValue [](start = 64, length = 26)

`baselineSharedPropertyValue`? Otherwise that variable isn't used.

But I'm confused because I thought you intended "shared" to mean "the same in the current run as it was in the baseline run".

Reply:
you're right in that baselineSharedPropertyValue is unused code that needs to be discarded. Done.

---
In reply to: [229080542](https://github.com/Microsoft/sarif-sdk/pull/1118#discussion_r229080542) [](ancestors = 229080542)